packages/client/examples/node.js/files.js

```diff
@@ -0,0 +1,46 @@
+import { Web3Storage } from '../../src/lib.js'
+import { Web3File } from 'web3-file'
+
+// TODO
+const endpoint = 'https://api.web3.storage' // the default
+const token = 'API_KEY' // your API key from https://web3.storage/manage
+
+async function main() {
+  const storage = new Web3Storage({ endpoint, token }) 
+
+  const files = prepareFiles()
+  const cid = await storage.put(files)
+
+  console.log('added', cid)
+}
+
+// TODO: Read a fixstures folder instead
+function prepareFiles () {
+  const data = 'Hello web3.storage!'
+  const data2 = 'Hello web3.storage!!'
+
+  return [
+    Web3File.fromBytes(
```

Comment:
Should we `Web3File.fromText()` https://github.com/web3-storage/web3-file/issues/5

Reply:
yey, just need to get it merged+shipped https://github.com/web3-storage/web3-file/pull/4 to update here